test/test_conjunctivegraph_operator_combinations.py

```diff
@@ -0,0 +1,123 @@
+import os
+from rdflib import (
+    Graph,
+    ConjunctiveGraph,
+    URIRef,
+)
+
+
+michel = URIRef("urn:x-rdflib:michel")
+tarek = URIRef("urn:x-rdflib:tarek")
+bob = URIRef("urn:x-rdflib:bob")
+likes = URIRef("urn:x-rdflib:likes")
+hates = URIRef("urn:x-rdflib:hates")
+pizza = URIRef("urn:x-rdflib:pizza")
+cheese = URIRef("urn:x-rdflib:cheese")
+
+c1 = URIRef("urn:x-rdflib:context-1")
+c2 = URIRef("urn:x-rdflib:context-2")
```

Comment:
```suggestion
michel = URIRef("urn:example:michel")
tarek = URIRef("urn:example:tarek")
bob = URIRef("urn:example:bob")
likes = URIRef("urn:example:likes")
hates = URIRef("urn:example:hates")
pizza = URIRef("urn:example:pizza")
cheese = URIRef("urn:example:cheese")

c1 = URIRef("urn:example:context-1")
c2 = URIRef("urn:example:context-2")
```

Reply:
Will do. Not actually forgotten, just currently distracted by trying to make sense of the chaos :smile: